client/src/pages/Application/FosterView/Step4.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Application (Foster View) Step 4
+ */
+
+import React from "react";
+import styled from "styled-components";
+import Meetings from "../../../components/Meeting";
+import logo from "../../../images/logo-inverted.png";
+import doggo from "../../../images/good-boi.png";
+import DogCard from "../../../components/DogProfileCard";
+
+const searchingForMatchesContent = (
+  <>
+    <p>
+      The TAP team is working hard to find the perfect foster match for you. Hang tight, while we
+      search.
+    </p>
+    <p>
+      Once we have found foster dogs that match your criteria they will be displayed to the right.
+      Let us know which ones you&apos;re interested in!
+    </p>
+  </>
+);
+
+const matchesFoundContent = (
+  <>
+    <p>The TAP team has found new foster matches!</p>
+    <p>View your matches to the right and let us know which ones you&apos;re interested in!</p>
+  </>
+);
+
+const FosterMatchesContainer = styled.div`
+  background: #fff;
+  border-radius: 15px;
+  padding: 2%;
+  height: 100%;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+`;
+
+const FosterMatchesContentContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  flex: 1;
+  height: 100%;
+  box-sizing: border-box;
+`;
+
+const DogsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  max-height: 400px;
+  overflow: scroll;
+  flex: 1;
+`;
+
+const Heading = styled.div`
+  font-size: 24px;
+  font-weight: 700;
+  font-size: 25px;
+`;
+
+const Green = styled.div`
+  color: #8dc442;
+  font-weight: 700;
+  font-size: 22px;
+`;
+
+function FosterMatches() {
+  const [matches, setMatches] = React.useState([
+    {
+      name: "tom",
+      imageRef: doggo,
+    },
+    {
+      name: "tom",
+      imageRef: doggo,
+    },
+
+    {
+      name: "tom",
+      imageRef: doggo,
+    },
+    {
+      name: "tom",
+      imageRef: doggo,
+    },
+  ]);
```

Comment:
Set this useState to an empty array (`[]`) to see the "Still looking for matches..." state